lib/cluster/zadara.rb

```diff
@@ -0,0 +1,81 @@
+module Cluster
+  class Zadara < Base
+
+    def self.all
+      resp_data = JSON.parse(api_request("GET", "vpsas.json").body)
+      resp_data["data"]
+    end
+
+    def self.find_vpsa
+      all.find do |vpsa|
+        vpsa["ip_address"] == storage_config[:nfs_server_host]
+      end
+    end
+
+    def self.hibernate
+      vpsa = find_vpsa
+      unless vpsa["name"].downcase.match("dev")
+        raise VpsaNotHibernatable.new(
+            "Refusing to hibernate vpsa '#{vpsa["name"]}': Not a 'dev' vpsa!")
```

Comment:
Are the dev vpsas created by chef recipes as part of the cluster creation that uses the name with 'dev' as part of the name? Is this feature only used for dev clusters? What about staging?

Reply:
No, the VPSAs still have to be created manually. Right now we only have one that is purposed for dev/testing and it's called "OCDEV". We'll most likely only ever have one, but a single vpsa storage pool can be carved up into multiple volumes which could potentially be used by different clusters. Right now it's only for staging. Again, see the email.